pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
+
+        // The upper limit for this quota.
+        int64 max_amount = 3;
```

Comment:
We just don't have the scoping story sorted out.

As it is, the quotasManager will be called with one of these config structs filled-in and will use max_amount as the limit for any quota of the given name. This means that the config will have been resolved to the appropriate <scope, subject>. But the code has no visibility to the hierarchy, so there's no way to enforce a single quota with limits at different levels of the hierarchy.

Similarly, we don't have a story for per-user overrides, beyond using different aspects for different users, which I think is lame.

Solving the hierarchical resolution and overrides in a clean way is a top concern post-Alpha. We need to have something that plays nice in this place that can clearly be understood by our operators and can be handled cleanly in scripting.